### PR TITLE
Created contributing.rst

### PR DESCRIPTION
**Pull request by tuckerrc**

### CODE REVIEW

File: contributing.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Contributing to QGIS Documentation
+**********************************
+
+Thank you for being willing to help with the QGIS Documentation.  If this is your first time checkout the guide found at http://docs.qgis.org/testing/en/docs/documentation_guidelines/first_contribution.html.
```
Comment (DelazJ) [inline contributing.rst]: Too much space before the word `If` :)
wonder if we should use this link or instead the global one at http://docs.qgis.org/testing/en/docs/documentation_guidelines/index.html
And to ease reviews, it's advised to shrink lines to around 80 characters.

**Comment by tuckerrc** [comment]
@DelazJ Thanks for the suggestions.  I made some changes to my branch.

File: CONTIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Contributing to QGIS Documentation
+=============
+
+Thank you for being willing to help with the QGIS Documentation.  If this is your
+first time checkout the [Documentation Guidlines][1].
```
Comment (DelazJ) [inline CONTIBUTING.md]: guidelines

Comment (tuckerrc) [reply CONTIBUTING.md]: Fixed

**Comment by DelazJ** [comment]
Nice @tuckerrc thanks

**Pull request merged**